### DISCUSSION TITLE
DIsable one minute red line overlaps

### DISCUSSION THREAD

**Discussion by gerroon** [kimai/kimai#3353] [Questions & Answers] [answered]
Hi,
Is there a way to disable the red lines that show up in the timesheets if there is an overlap? I like the feature but it shows up even with one min overlaps which is useless to me. I use multiple client devices with multiple browsers and they generally have the same time but can have a min difference, some use ntp some do not, some other strict firewall issues also block time servers etc.

What I am wondering if there is a way to disable it or increase the overlap period so that my timesheets is not littered with red lines. I fix them manually by bumping up the mins but I prefer to just fire and forget time recording.

This issue happens when I start a new activity from the last activities top corner menu, which stops the current timer and starts another activity.

Thanks

**Comment by kevinpapst** [comment, marked as answer]
Choose other rounding config. 

This system configuration for example will stop the problem and you won't be able to trigger the overlapping red line indicator issue by using the restart button anymore: 
<img width="449" alt="Bildschirmfoto 2022-06-09 um 12 15 16" src="https://user-images.githubusercontent.com/533162/172824038-abcb09ae-e6ef-4847-b506-e50e3318795d.png">

**Comment by gerroon** [comment]
Hi

This was working fine but I still got one today. Is there a way to extend the overlap maybe?

![chrome_1f5sxTvIJl](https://user-images.githubusercontent.com/8519469/173697598-d77a87fd-fe0f-4056-9f21-938d70a2d995.jpg)

**Reply by kevinpapst** [reply]
Try "floor" or "ceil" rounding and use 1 instead of 0, to get rid of seconds in your records.
Other option is to install the Custom CSS bundle and hide the line
